examples/fixture/asm/kimchi/public_output_bool.asm

```diff
@@ -0,0 +1,29 @@
+@ noname.0.7.0
+
+DoubleGeneric<1>
+DoubleGeneric<1>
+DoubleGeneric<1,1,-1>
+DoubleGeneric<1,0,0,0,-2>
+DoubleGeneric<1,0,-1,0,6>
+DoubleGeneric<1,0,0,0,-8>
+DoubleGeneric<1,1>
+DoubleGeneric<1,1,-1>
+DoubleGeneric<0,0,-1,1>
+DoubleGeneric<1,1>
+DoubleGeneric<1,0,-1,0,1>
+DoubleGeneric<1,-1>
+DoubleGeneric<0,0,-1,1>
+DoubleGeneric<1>
+DoubleGeneric<1,-1>
+(0,0) -> (14,0)
```

Comment:
public output is the first public input (by convention), which is wired to the last gate, makes sense to me if squinting